### PR TITLE
@types/joi add dataUri type definition

### PR DESCRIPTION
**Pull request by TheWillG**
Joi v13.4.0 introduced string().dataUri()

### CODE REVIEW

**Comment by SimonSchick** [comment]
@TheWillG please fill out the template next time.

**Comment by SimonSchick** [comment]
Any chance you can also add the `Map` and `Symbol` rules?

**Comment by TheWillG** [comment]
I'm not sure I understand what you are referring to in regards to `Map` and `Symbol`. Is this in the Joi API?

**Review by SimonSchick — APPROVED** [review]

**Comment by SimonSchick** [comment]
Ah yea, it's just `Symbol`, the `Map` type wasn't added just a rule called `map`.

**Comment by SimonSchick** [comment]
Although, these are technically not released, so I guess this is fine.

**Comment by TheWillG** [comment]
Alright, sounds good. I'll just leave this as-is. Thanks.

File: types/joi/index.d.ts [hunk 1]
```diff
@@ -147,6 +148,13 @@ export interface UriOptions {
     relativeOnly?: boolean;
 }
 
+export interface DataUriOptions {
+    /**
+     * optional parameter defaulting to true which will require = padding if true or make padding optional if false
+     */
+    paddingRequired?: boolean;
```
[the remark this answers is not in the document]
Comment (TheWillG) [reply types/joi/index.d.ts]: Isn't that what the comment on L153 is for?

[the remark this answers is not in the document]
Comment (SimonSchick) [reply types/joi/index.d.ts]: Derp, my bad, the way to document default values for interfaces varies a lot, I skipped it when reading 😛

**Comment by SimonSchick** [comment]
Hey @TheWillG minor feature creep, I noticed I forgot myself to the authors of this package, would you mind adding me there? `Simon Schick <https://github.com/SimonSchick>`

**Comment by TheWillG** [comment]
Done

**Pull request merged**